Apps/WebClient/src/NewClientApp/src/models/dateWrapper.ts

```diff
@@ -2,7 +2,7 @@ import {
     DateTime,
     DateTimeUnit,
     Duration,
-    DurationLikeObject,
+    DurationLike,
```

Comment:
This is interesting Performed the conversion and `DurationLike` wasn't an available type, have they reintroduced it or was there another way to implement it?

I would have done this on Rider for consideration.

Reply:
I think we were using an old version of `@types/luxon` previously that didn't export the type.  The current one does!